generator/templates/static/Graph.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Spatie\SchemaOrg;
+
+use ReflectionClass;
+use BadMethodCallException;
+use InvalidArgumentException;
+
+/**
+ * @mixin \Spatie\SchemaOrg\Schema
+ */
+class Graph extends BaseType
```

Comment:
I made some minor code style changes to this class. Could you rerun the generator and commit to update the file in `src/`?